assets/xml/objects/object_in.xml

```diff
@@ -4,114 +4,124 @@
-        <DList Name="object_in_DL_011AD8" Offset="0x11AD8"/>
-        <DList Name="object_in_DL_012120" Offset="0x12120"/>
-        <DList Name="object_in_DL_012220" Offset="0x12220"/>
-        <DList Name="object_in_DL_012350" Offset="0x12350"/>
-        <DList Name="object_in_DL_012480" Offset="0x12480"/>
-        <DList Name="object_in_DL_0125B0" Offset="0x125B0"/>
-        <DList Name="object_in_DL_0126E0" Offset="0x126E0"/>
-        <DList Name="object_in_DL_012AC0" Offset="0x12AC0"/>
-        <DList Name="object_in_DL_012BF0" Offset="0x12BF0"/>
-        <DList Name="object_in_DL_012D20" Offset="0x12D20"/>
-        <DList Name="object_in_DL_013100" Offset="0x13100"/>
-        <DList Name="object_in_DL_0132B8" Offset="0x132B8"/>
-        <DList Name="object_in_DL_0133F0" Offset="0x133F0"/>
-        <DList Name="object_in_DL_013540" Offset="0x13540"/>
-        <DList Name="object_in_DL_013688" Offset="0x13688"/>
-        <DList Name="object_in_DL_0137C0" Offset="0x137C0"/>
-        <DList Name="object_in_DL_013910" Offset="0x13910"/>
-        <Limb Name="object_in_Limb_013A58" LimbType="Standard" Offset="0x13A58"/>
-        <Limb Name="object_in_Limb_013A64" LimbType="Standard" Offset="0x13A64"/>
-        <Limb Name="object_in_Limb_013A70" LimbType="Standard" Offset="0x13A70"/>
-        <Limb Name="object_in_Limb_013A7C" LimbType="Standard" Offset="0x13A7C"/>
-        <Limb Name="object_in_Limb_013A88" LimbType="Standard" Offset="0x13A88"/>
-        <Limb Name="object_in_Limb_013A94" LimbType="Standard" Offset="0x13A94"/>
-        <Limb Name="object_in_Limb_013AA0" LimbType="Standard" Offset="0x13AA0"/>
-        <Limb Name="object_in_Limb_013AAC" LimbType="Standard" Offset="0x13AAC"/>
-        <Limb Name="object_in_Limb_013AB8" LimbType="Standard" Offset="0x13AB8"/>
-        <Limb Name="object_in_Limb_013AC4" LimbType="Standard" Offset="0x13AC4"/>
-        <Limb Name="object_in_Limb_013AD0" LimbType="Standard" Offset="0x13AD0"/>
-        <Limb Name="object_in_Limb_013ADC" LimbType="Standard" Offset="0x13ADC"/>
-        <Limb Name="object_in_Limb_013AE8" LimbType="Standard" Offset="0x13AE8"/>
-        <Limb Name="object_in_Limb_013AF4" LimbType="Standard" Offset="0x13AF4"/>
-        <Limb Name="object_in_Limb_013B00" LimbType="Standard" Offset="0x13B00"/>
-        <Limb Name="object_in_Limb_013B0C" LimbType="Standard" Offset="0x13B0C"/>
-        <Limb Name="object_in_Limb_013B18" LimbType="Standard" Offset="0x13B18"/>
-        <Limb Name="object_in_Limb_013B24" LimbType="Standard" Offset="0x13B24"/>
-        <Limb Name="object_in_Limb_013B30" LimbType="Standard" Offset="0x13B30"/>
-        <Skeleton Name="object_in_Skel_013B88" Type="Flex" LimbType="Standard" Offset="0x13B88"/>
+        <Texture Name="gIngoShirtTLUT" OutName="shirt_tlut" Format="rgba16" Width="16" Height="16" Offset="0x1CD0"/>
+        <Texture Name="gIngoShoulderTLUT" OutName="shoulder_tlut" Format="rgba16" Width="16" Height="16" Offset="0x1ED0"/>
```

Comment:
```suggestion
        <Texture Name="gIngoSkinGradientTLUT" OutName="skin_gradient_tlut" Format="rgba16" Width="16" Height="16" Offset="0x1ED0"/>
```
I think, since it only gets used for the skin gradient